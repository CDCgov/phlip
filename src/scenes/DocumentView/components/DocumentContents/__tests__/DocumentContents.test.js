import React from 'react'
import { shallow } from 'enzyme'
import { DocumentContents } from '../index'

const props = {
  document: {
    content: {},
    name: 'Test file'
  },
  loading: false
}

describe('DocumentView - DocumentContents', () => {
  test('should render correctly', () => {
    expect(shallow(<DocumentContents {...props} />)).toMatchSnapshot()
  })
})