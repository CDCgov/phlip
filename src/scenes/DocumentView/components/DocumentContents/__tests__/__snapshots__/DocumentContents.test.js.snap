// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DocumentView - DocumentContents should render correctly 1`] = `
<FlexGrid
  align="stretch"
  circular={false}
  container={true}
  flex={true}
  justify="stretch"
  padding={0}
  raised={true}
  style={
    Object {
      "flexBasis": "70%",
      "minWidth": "65%",
      "overflow": "hidden",
      "padding": 20,
    }
  }
  type="column"
>
  <FlexGrid
    align="stretch"
    circular={false}
    container={true}
    flex={false}
    justify="stretch"
    padding={0}
    raised={false}
    style={
      Object {
        "display": "inline-flex",
        "marginBottom": 10,
        "position": "relative",
      }
    }
    type="column"
  >
    <WithStyles(Typography)
      style={
        Object {
          "color": "#9e9e9e",
          "fontSize": ".65rem",
          "marginBottom": 2,
        }
      }
      variant="caption"
    >
      Document Name
    </WithStyles(Typography)>
    <WithStyles(Typography)
      variant="subheading"
    >
      Test file
    </WithStyles(Typography)>
  </FlexGrid>
</FlexGrid>
`;
